gax-java/gax/src/main/java/com/google/api/gax/tracing/BaseApiTracer.java

```diff
@@ -104,10 +106,17 @@ public void attemptCancelled() {
   }
 
   @Override
-  public void attemptFailed(Throwable error, Duration delay) {
+  public void attemptFailedDuration(Throwable error, java.time.Duration delay) {
     // noop
   }
 
+  /** Backport of {@link #attemptFailedDuration(Throwable, java.time.Duration)} */
+  @Override
+  @ObsoleteApi("Use attemptFailed(Throwable, java.time.Duration) instead")
+  public void attemptFailed(Throwable error, org.threeten.bp.Duration delay) {
+    attemptFailedDuration(error, toJavaTimeDuration(delay));
+  }
```

Comment:
Probably no-op here as well. Might be missing some details, but I don't see any benefit of having to convert the time to call the other method.

Reply:
Done